Code/GraphMol/MolDraw2D/Wrap/rdMolDraw2D.cpp

```diff
@@ -62,7 +62,21 @@ std::map<int, double> *pyDictToDoubleMap(python::object pyo) {
   }
   return res;
 }
+
+void pyListToColourVec(python::object pyo, std::vector<DrawColour> &res) {
+  python::list tList = python::extract<python::list>(pyo);
+  for (unsigned int i = 0;
+       i < python::extract<unsigned int>(tList.attr("__len__")()); ++i) {
+    python::tuple tpl = python::extract<python::tuple>(tList[i]);
+    float r = python::extract<float>(tpl[0]);
```

Comment:
it's too easy to make the obvious mistake here and provide the colours as values between 0 and 255 instead of 0 and 1. you should do a range check on each component and raise a `ValueError` if it's less than 0 or greater than 1.